biojava-structure/src/main/java/org/biojava/nbio/structure/chem/ChemComp.java

```diff
@@ -0,0 +1,406 @@
+package org.biojava.nbio.structure.chem;
+
+import org.biojava.nbio.structure.io.cif.CifBean;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Properties of a chemical component.
+ * @author Sebastian Bittrich
+ * @since 6.0.0
+ */
+public class ChemComp implements CifBean, Comparable<ChemComp> {
```

Comment:
They are not needed per se. My motivation was that e.g. `ChemComp` is exposed via `Group` and I wanted to avoid exposing access methods coming from ciftools (having to call `#getId().get(0)`). Also, these beans have some additional fields/functionality, e.g. `ChemComp` stores associated atoms and bonds and there are inferred fields like `residueType`, `polymerType`, and `standard`.

Certainly open for debate. I'd like to get rid of all duplicates (i.e., everything annotated as `CifBean`). But this is the best initial solution I saw.

Reply:
Ok, thanks for explaining. I think it is a great solution, at least for the time being.